### PR TITLE
Moved layout code to separate classes 2

### PR DESCRIPTION
**Pull request by igorkorsukov**
Just moved code, no logic changes

Step by step

### CODE REVIEW

**Review by Eism — APPROVED** [review]

File: src/engraving/libmscore/textbase.cpp [hunk 1]
```diff
@@ -2430,7 +2430,7 @@ void TextBase::writeProperties(XmlWriter& xml, bool writeText, bool /*writeStyle
     }
 }
 
-static constexpr std::array<Pid, 18> pids { {
+static constexpr std::array<Pid, 18> text_pids { {
```
Comment (Jojo-Schmitz) [inline src/engraving/libmscore/textbase.cpp:2433]: conflicts with #8798, there other names have been proposed by @RomanPudashkin

Comment (Jojo-Schmitz) [reply src/engraving/libmscore/textbase.cpp:2433]: `TextBasePropertyId` actually.

Comment (Jojo-Schmitz) [reply src/engraving/libmscore/textbase.cpp:2433]: Thanks for merging that. Now this here needs a rebase

**Review by Jojo-Schmitz — APPROVED** [review]
Should esp. avoid the issue mentioned in https://github.com/musescore/MuseScore/pull/8888#issuecomment-902548711

**Pull request merged**